tests/dragonfly/replication_test.py

```diff
@@ -1836,3 +1836,46 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     await c_replica.close()
     master.stop()
     replica.stop()
+
+
+@pytest.mark.asyncio
+@pytest.mark.slow
+async def test_eviction_propagation(df_local_factory, df_seeder_factory):
+    master = df_local_factory.create(proactor_threads=1, cache_mode="true", maxmemory="290mb")
+    replica = df_local_factory.create(proactor_threads=1)
+    df_local_factory.start_all([master, replica])
+
+    c_master = master.client()
+    c_replica = replica.client()
+
+    seeder = df_seeder_factory.create(port=master.port, val_size=1000, stop_on_failure=False)
+
+    await c_master.execute_command("DEBUG POPULATE 6200000")
+    await c_replica.execute_command(f"REPLICAOF localhost {master.port}")
+    await wait_available_async(c_replica, timeout=3000)
+
+    # Check data after full sync
+    await check_all_replicas_finished([c_replica], c_master)
+    await check_data(seeder, [replica], [c_replica])
+
+    # Stream more data in stable state
+    await seeder.run(target_ops=10000000, target_deviation=0.0001)
```

Comment:
wow thats a lot of ops, I dont think we can run this test in regression test this will take forever
Also you dont need to send target_deviation when sending target_ops
when you run with target_ops the seeder will run this ops, when you run seeder with number of keys and  target_deviation it will fill the database with the number of keys * (1-target_deviation)

Reply:
ha! actually no... let me change it to a smaller value... i think a few thousand will do. 

the target_deviatoin does help ... with that one it seems I actually only executed a couple thousand ops... or even less than that although target_ops is a huge number...